### DISCUSSION TITLE
Cannot access storagepool directory

### DISCUSSION THREAD

**Discussion by eriksmits80** [trapexit/mergerfs#1138] [Q&A] [answered]
**The problem**
I tried to merge two drives (mounted as /mnt/drive1 and /mnt/drive2) into a pool on /mnt/storage. When I mount the pool and try to access the directory I get the message "no such file or directory". When I try to list the content of /mnt i get the message "cannot access 'storage' :  Permission denied". The list is then as follows

ls: cannot access '/mnt/storage': Permission denied
total 8
drwxrwxr-x 4 erik erik 4096 Jan 27 09:40 disk1
drwxrwxr-x 4 erik erik 4096 Feb  6 14:43 disk2
d????????? ? ?    ?       ?            ? storage

**What I did so far**
I installed the server last week and everything should be up to date.
Mergerfs is installed this morning as per instructions using [mergerfs_2.34.1.debian-bullseye_amd64.deb]. 
In /etc/fstab I added
`/mnt/drive*     /mnt/storage    fuse.mergerfs   cache.files=partial,dropcacheonclose=true,category.create=mfs    0       0`
(

I have been searching github and google, but have no clue what causes the problem. Then again, I am quite new to linux, so am probably missing something. My first thought was something to do with permissions, but no matter what I try (changing permissions, change owner) I get the same result. Logging in as root and trying to acces /mnt/storage also tells me that is does not exist, eventhough it is listed. Unmounting gives me access to the folder, but that doen't really solve the problem, right. So, I'm kind of empty on idea's so I hope someone can point me in the right direction. Thanks in advance.

**Comment by trapexit** [comment]
You don't have `allow_other` argument. 

https://github.com/trapexit/mergerfs/tree/2.34.1#basic-setup

You have to use the documentation for the version you're using. You likely are looking at the latest docs for a not yet released version where I remove the need for `allow_other` and `use_ino`.

**Comment by eriksmits80** [comment, marked as answer]
I have now changed /etc/fstab to
`/mnt/drive*     /mnt/storage    fuse.mergerfs   allow_other,use_ino,cache.files=off,dropcacheonclose=true,category.create=mfs   0       0
`
as in the link mentioned but still the same result unfortunately


/edit: location and mention in fstab were not the same.  Fixing this solved the problem.

**Reply by trapexit** [reply]
And you remounted the filesystem?

**Reply by eriksmits80** [reply]
Yes, remounted the filesystem. Also tried if restarting the system would work. Just in case.

**Reply by trapexit** [reply]
Are you positive you have the branch list correct? You show your fstab entry to have `/mnt/drive*` yet in the original post you show /mnt/disk1 and /mnt/disk2.

**Reply by trapexit** [reply]
It's coming in a future release that it will at least warn about this situation but having a currently not existent paths is a valid usecase.

**Reply by eriksmits80** [reply]
I guess I got blind staring at text all day. I indeed used disk in the branch list and drive in the fstab entry. It seems to be working. Many thanks for your help.
